elide-model-config/src/main/java/com/yahoo/elide/modelconfig/model/Table.java

```diff
@@ -68,10 +66,10 @@
     private String dbConnectionName;
 
     @JsonProperty("isFact")
-    private Boolean isFact = true;
+    private Boolean isFact;
```

Comment:
@aklish are this fields purposely modifiable after building?  Or should they all be final?
```suggestion
    private final Boolean isFact;
```

Reply:
I think we cannot declare all fields final, because we have inheritance logic